ui/app/adapters/application.js

```diff
@@ -92,7 +92,7 @@ export default RESTAdapter.extend({
         controlGroup.deleteControlGroupToken(controlGroupToken.accessor);
       }
       const [resp] = args;
-      if (resp && resp.warnings) {
+      if (resp && resp.warnings && !options.skipWarnings) {
```

Comment:
I might be missing it, but if we don't have test coverage for this, we should add it. Test coverage can sometimes act as documentation for expected behavior. So for someone not understanding what warnings we're skipping a test with sample warnings can better explain it.

Reply:
Good call! There actually wasn't an existing test for this adapter 😱 but I went ahead and added one